autoPyTorch/api/base_task.py

```diff
@@ -657,9 +649,11 @@ def _do_traditional_prediction(self, num_run: int, time_left: int, func_eval_tim
                             f"Fitting {cls} took {runtime}s, performance:{cost}/{additional_info}")
                         configuration = additional_info['pipeline_configuration']
                         origin = additional_info['configuration_origin']
+                        additional_info.pop('pipeline_configuration')
```

Comment:
Sorry, why are you doing pop here?

Reply:
so we dont have pipeline configuration in the run info as well as additional info